### PR TITLE
Add troubleshooting solution for very large projects

### PR DESCRIPTION
**Pull request by geealbers**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

For [Discussion #](https://github.com/thegetty/quire/discussions/742)

### Please describe the goal of this pull request and the changes that were made.

Add troubleshooting steps for addressing the node memory issue that occurs in very large Quire projects

### CODE REVIEW

File: content/docs-v1/troubleshooting.md [hunk 1]
```diff
@@ -23,4 +23,38 @@ There is currently an issue with `quire build` on Windows 10. It is with one of
...
 
-If you're a Mac user and receive an error message that references "Developer/CommandLineTools" that means you need to install Apple Xcode, a set of developer tools for your Mac. Run the command `xcode-select --install`. It may take a few minutes to install. Once install is complete, try creating a new project again. 
\ No newline at end of file
+If you're a Mac user and receive an error message that references "Developer/CommandLineTools" that means you need to install Apple Xcode, a set of developer tools for your Mac. Run the command `xcode-select --install`. It may take a few minutes to install. Once install is complete, try creating a new project again. 
+
+## Other Issues
+
+### Working with Very Large Projects
+
+The following error has been seen in projects with a lot of individual Markdown files, a lot of large images, or a lot of shortcode usage. It is caused by node.js running out of memory while trying to process the project files.
```
Comment (Erin-Cecele) [inline content/docs-v1/troubleshooting.md:32]: Node.js

File: content/docs-v1/troubleshooting.md [hunk 2]
```diff
@@ -23,4 +23,38 @@ There is currently an issue with `quire build` on Windows 10. It is with one of
...
+
+    Windows:
+
+    ```
+    $env:NODE_OPTIONS = '--max-old-space-size=12288'
+    ```
+
+4. Run `quire preview`
+
+If the error persists try allocating more memory by repeating Step 3 with a larger number. Up to half a GB less than your total memory.
```
Comment (Erin-Cecele) [inline content/docs-v1/troubleshooting.md:58]: This is the maximum right, so if I have 64 GB than the max would be 63.5?

File: content/docs-v1/troubleshooting.md [hunk 3]
```diff
@@ -23,4 +23,38 @@ There is currently an issue with `quire build` on Windows 10. It is with one of
...
+
+**Note:** setting the `NODE_OPTIONS` in this way will only persist as long as you have the same command-line session running. You will need to use the above `export`/`$env` command each time you start a close and re-open the command-line shell to work on your large project.
```
Comment (Erin-Cecele) [inline content/docs-v1/troubleshooting.md:60]: each time you close and re-open (I think "start a" is a typo/leftover from a previous edit)

**Review by Erin-Cecele — APPROVED** [review]
I accidentally submitted one other comment as a single comment. That was just a matter of capitalization. Aside from small edits and one question, this looks good to go, so I'm approving.

**Comment by geealbers** [comment]
Thanks @Erin-Cecele, I've made those changes/clarifications and am merging it now.

**Pull request merged**